### PR TITLE
Styling/planet cards

### PR DESCRIPTION
**Pull request by ConnorAndersonLarson**
# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [X] Fix
- [ ] Feature
- [X] Refactor
- [X] Style
- [ ] Testing

### Are there any known bugs?

- [X] No
- [ ] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
Adds a prompt to press buttons to increase usability. Also adds unique scaling based on actual planet size (not to scale)

### What ticket(s) do the changes resolve?

Please Link Issues to this PR as necessary.

### Notes for the Reviewer
The was done in App.tsx and Planet.tsx/.css

### How to Test Changes?
open branch, run npm start, observe planets on hover.

### CODE REVIEW

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -84,6 +84,7 @@ class App extends React.Component<{}, AllData> {
                   <SortBox updateSort={this.updateSort} />
                   {this.state.error && <h2>{this.state.error}</h2>}
                   {!this.state.error && !this.state.allPlanets.length && <h2>Loading...</h2>}
+                  {!this.state.error && !!this.state.allPlanets.length && <h2>Click on a planet to learn more!</h2>}
```
Comment (alexthompson207) [inline src/components/App/App.tsx:87]: !! is weird but I glad you got it to work!

Comment (knees4bees) [reply src/components/App/App.tsx:87]: ^^^

File: src/components/Planet/planet.css [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .planet-card-name, .planet-card-fact{
-  transition: color 1s
+  transition: color .5s
```
Comment (alexthompson207) [inline src/components/Planet/planet.css:2]: I like the speedier transition, good work!

File: src/components/Planet/planet.css [hunk 3]
```diff
@@ -17,11 +17,47 @@
...
+
+.Uranus-img:hover {
+  transform: scale(1.10);
+  transition: transform .5s ease-in-out;
+}
+
+.Neptune-img:hover {
   transform: scale(1.05);
-  transition: transform 1s ease-in-out;
+  transition: transform .5s ease-in-out;
```
Comment (alexthompson207) [inline src/components/Planet/planet.css:57]: Same here!

File: src/components/Planet/planet.css (hunk 2 above)
Comment (knees4bees) [inline src/components/Planet/planet.css:2]: This is more natural now. Nice!

**Review by alexthompson207 — APPROVED** [review]

File: src/components/Planet/planet.css [hunk 4]
```diff
@@ -17,11 +17,47 @@
...
+  transform: scale(1.15);
+  transition: transform .5s ease-in-out;
+}
+
+.Uranus-img:hover {
+  transform: scale(1.10);
+  transition: transform .5s ease-in-out;
+}
+
+.Neptune-img:hover {
```
Comment (knees4bees) [inline src/components/Planet/planet.css:55]: Love that the effect is a different size depending on the size of the planet. Very cool idea!!

**Review by knees4bees — APPROVED** [review]

**Pull request merged**